app/views/projects/show.html.erb

```diff
@@ -8,36 +8,60 @@ Project Details:
     Your new project request is in the queue. Please allow 5 business days for our team to review your needs and set everything up. For assistance, please contact tigerdata@princeton.edu.
   </div>
 <% end %>
-<div>
+
+<div class="container-fluid">
   <h2>Roles</h2>
   <dl>
-  <dt>Data Sponsor</dt> <dd><%= @project.metadata[:data_sponsor] %></dd>
-  <dt>Data Manager</dt> <dd> <%= @project.metadata[:data_manager] %></dd>
+    <% if !@data_sponsor.nil? %>
+      <dt>Data Sponsor</dt>
+      <dd>
+        <%= @data_sponsor.given_name %> <%= @data_sponsor.family_name %>
+      </dd>
+    <% end %>
+    <% if !@data_manager.nil? %>
+      <dt>Data Manager</dt>
+      <dd>
+        <%= @data_manager.given_name %> <%= @data_manager.family_name %>
+      </dd>
+    <% end %>
 
-  <dt>Read Only</dt> <dd>
-    <%if @project.metadata[:data_user_read_only].present? %>
-      <% @project.metadata[:data_user_read_only].each do |user|%>
-       <%= user%> 
-      <%end%>
+    <dt>Read Only</dt>
+    <% if @data_read_only_users.present? %>
+      <dd>
+        <% @data_read_only_users.each_with_index do |user, idx|%>
+          <% if user.given_name.blank? && user.family_name.blank? %>
+            <%= user.uid %><%= idx < (@data_read_only_users.length - 1) ? ', ' : '' %>
+          <% else %>
+            <%= user.given_name %> <%= user.family_name %><%= idx < (@data_read_only_users.length - 1) ? ', ' : '' %>
+          <% end %>
+        <% end %>
       </dd>
-    <%else%>
```

Comment:
@jrgriffiniii you are removing required functionality here: https://github.com/pulibrary/tiger-data-app/issues/266